jerry-core/debugger/debugger.c

```diff
@@ -13,8 +13,6 @@
  * limitations under the License.
  */
 
-#define _XOPEN_SOURCE 500 /* Required macro for sleep functions */
```

Comment:
The only change that's necessary is to move this definition from this source ...